packages/core/src/controllers/RouterController.ts

```diff
@@ -185,6 +189,9 @@ export const RouterController = {
   },
 
   goBack() {
+    const shouldReload =
```

Comment:
Would it make sense to set this value inside the setTimeout in case of potential closure issues? Defining the variable outside might lead to problems if the value of `shouldReload` is modified elsewhere in the code before the setTimeout executes

Reply:
If we would add in the setTimeout, the `this.state.view` is not accurate anymore, so that is why I added this before. The value of `shouldReload` won't change in the code below.